### DISCUSSION TITLE
Repository not found when running mage build

### DISCUSSION THREAD

**Discussion by emcfins** [aquasecurity/trivy#7091] [Development]
I'm trying to build trivy locally but it looks like I'm running into a snag. I'm following the directions on this [page](https://aquasecurity.github.io/trivy/v0.53/community/contribute/pr/) but when I go to run `mage build`, I'm getting an error:

```
	github.com/mitchellh/osext@v0.0.0-20151018003038-5e2d6d41470f: invalid version: git ls-remote -q origin in /Users/<snip>/go/pkg/mod/cache/vcs/94ed57c5b21c953d93b47487113db43a5c9b69fd990329ec70dc77348c4dd443: exit status 128:
	ERROR: Repository not found.
	fatal: Could not read from remote repository.

	Please make sure you have the correct access rights
	and the repository exists.
```

It's been able to pull from other repositories until it reaches this one. Not sure what I'm doing wrong but any pointer in the right direction would be greatly appreciated.

**Comment by knqyf263** [comment]
It seems like the repository has been removed.
https://github.com/mitchellh/osext

It can be downloaded via Go proxy.

`GOPROXY=https://proxy.golang.org,direct`

https://proxy.golang.org/github.com/mitchellh/osext/@v/v0.0.0-20151018003038-5e2d6d41470f.info
